<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tasks_panel</class>
 <widget class="QWidget" name="tasks_panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasks tracking</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="tasks_list">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Task tracking</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="add_task_btn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="edit_task_btn">
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/icons/icons/edit.png</normaloff>:/icons/icons/edit.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="del_task_btn">
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/icons/icons/remove.png</normaloff>:/icons/icons/remove.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="execute_task_btn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/icons/icons/media-playback-start.png</normaloff>:/icons/icons/media-playback-start.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="create_task_action">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>Create task</string>
   </property>
   <property name="toolTip">
    <string>Create the new task</string>
   </property>
  </action>
  <action name="delete_task_action">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/icons/remove.png</normaloff>:/icons/icons/remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete task</string>
   </property>
   <property name="toolTip">
    <string>Delete selected task</string>
   </property>
  </action>
  <action name="execute_task_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/icons/media-playback-start.png</normaloff>:/icons/icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Execute selected task</string>
   </property>
   <property name="toolTip">
    <string>Execute selected task</string>
   </property>
  </action>
  <action name="edit_task_action">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/icons/edit.png</normaloff>:/icons/icons/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit current task</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
